packages/ra-ui-materialui/src/detail/TabbedShowLayout.js

```diff
@@ -87,17 +87,19 @@ const TabbedShowLayout = props => {
         ...rest
     } = props;
     const match = useRouteMatch();
-
     const classes = useStyles(props);
+    const finalChildren = Array.isArray(children)
```

Comment:
Use React.Children.toArray, because the children prop data structure isn't documented and may change in the future. 

Also, I'd call the variable `nonNullChildren`